.whitesource

```diff
@@ -0,0 +1,15 @@
+{
+  "scanSettings": {
+    "configMode": "AUTO",
+    "configExternalURL": "",
+    "projectToken": "",
+    "baseBranches": []
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure",
+    "displayMode": "diff"
+  },
+  "issueSettings": {
+    "minSeverityLevel": "LOW"
```

Comment:
Trying to understand, @bbarani looks like the bot opens up issues in public if `minServerityLevel` is not NONE.
Do we really want all the security vulnerabilities reported publicly ?

Reply:
Never mind just saw your comment. Makes sense!